File: WebAPI/WebAPI/Models/Review.cs
namespace WebAPI.Models
{
    public class Review
    {
        public Guid ReviewId { get; set; }
        public Guid UserId { get; set; }
        public Guid ProductId { get; set; }
        public int Rating { get; set; }
        public string Comment { get; set; }
        public DateTime ReviewDate { get; set; }

        public User User { get; set; }
        public Product Product { get; set; }
    }
}

File: UI/PhoneShop/src/app/core/components/product-detail/models/review.ts
export interface Review {
    reviewId: string;
    userId: string;
    userName: string; // Thông tin người dùng (ví dụ: tên người dùng)
    productId: string;
    productName: string;// Thông tin sản phẩm (ví dụ: tên sản phẩm)
    rating: number;
    comment: string;
    reviewDate: string;
}
